packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -31,8 +32,7 @@ export const SimpleDocument: FC<Props> = ({
   document,
   highlight,
   setLoading,
-  setHideToolbarControls,
-  cannotPreviewMessage = 'Cannot preview document'
```

Comment:
Like here: `cannotPreviewMessage` is the overrideable (sp?) "i18n" message. You'll need to add another one for the sub-heading.